src/qml/BaseMovesComponent.qml

```diff
@@ -25,10 +25,13 @@ import org.kde.kirigami 2.5 as Kirigami
 import org.thetailcompany.digitail 1.0
 
 Item {
-    id: root;
-    signal commandActivated(string command);
-    property bool blockOnMovingTail: false;
+    id: control;
```

Comment:
This feels jarring to me - if it's the component's root item, please call it root

Reply:
Each component in each file should have id `control`, not `root`, it is useful to reserve `root` id for real root, in our case it is `main.qml`. So, if we call each item as `root` we can not access to real `root`.

Reply:
Off course, we can rename `root` to `rootItem`, but it is not a good solution

Reply:
Really, i have no reason for calling it "root" other than that being what i've always done - but being as that is what i've always done, that's how the rest of the codebase works... Don't change it in this single component, it's quite badly out of scope for this patch...

Reply:
In principle, i've not really got anything against it being changed, but as it's a change in general policy, it really wants to be done everywhere at the same time rather than piecemeal

Reply:
So, I got you, and maybe we should rename all not real `root` components to `control` later?

Reply:
Yeah, later is good :)